### ISSUE TITLE
MemoizedSelector needs to be exported as well

### ISSUE THREAD

**Issue by alex-okrushko** [ngrx/platform#49]
export const selectFeature = createFeatureSelector<FeatureState>('feature');
                ~~~~~~~~~~~~~~~

error TS4023: Exported variable 'selectFeature' has or is using name 'MemoizedSelector' from external module ".../ngrx/modules/store/src/selector" but cannot be named.

**Comment by KaDee** [comment]
+1. Yes, please.

**Comment by brandonroberts** [comment]
Closed via #57
